onnx/reference/ops/op_affine_grid.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) ONNX Project Contributors
+
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=R0912,R0913,R0914,R0915,R1702,R1716,W0221
+
+import numpy as np
+
+from onnx.reference.op_run import OpRun
+
+
+def construct_original_grid(data_size, align_corners):
+    is_2d = len(data_size) == 2
+    size_zeros = np.zeros(data_size)
+    original_grid = [np.ones(data_size)]
+    for dim, dim_size in enumerate(data_size):
+        assert dim_size > 0

```

Comment:
If this test is only for debugging, it should be removed, otherwise, a better error message would be helpful.

Reply:
done